src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -359,7 +359,7 @@ export default function myIntegration() {
 
 #### `routes` option
 
-**Type:** [`RouteData[]`](https://github.com/withastro/astro/blob/main/packages/astro/src/%40types/astro.ts#L973)
+**Type:** [`RouteData[]`](https://github.com/withastro/astro/blob/main/packages/astro/src/%40types/astro.ts#L1525)
```

Comment:
Thanks for this @MichaelJolley!

This line number is going to change every time that file is edited so I’m wondering if a link here is the best idea in the first place. Given we include the detailed type reference in this section, maybe we don’t need the link? We could also make the “`RouteData` type reference” heading an `<h5>` instead of just bold and link to that?

Reply:
We can also use a permalink: https://github.com/withastro/astro/blob/6c465e958e088ff55e5b895e67c64c0dfd4277a6/packages/astro/src/%40types/astro.ts#L1525-L1537